src/ansys/fluent/core/services/settings.py

```diff
@@ -259,11 +259,20 @@ def _extract_static_info(self, info: SettingsModule.StaticInfo) -> dict[str, Any
                 child.name: self._extract_static_info(child.value)
                 for child in info.arguments
             }
+            try:
+                for child in info.arguments:
+                    ret["arguments"][child.name].update(
```

Comment:
I thought already we had done check for arguments and we are adding file_purpose on argument level so that we can just update the arguments with respective file purpose.

Reply:
We get following error if we check above condition.

![image](https://github.com/ansys/pyfluent/assets/106588300/1d1948c1-1388-4633-9e6e-e30cf859a346)

Reply:
What is special about attrs? - `attrs` is not a separate field in static info. 

Can't you do in the outer block - It's inside arguments therefore we are doing in arguments block.

Reply:
The `"type"` and `"help"` are also added for arguments, so `"attrs"` should also be added at the same level. We'll be already inside the arguments while adding those in the inner call.

Reply:
Done. Thank you.